test/integration/pj-rehearse/expected.yaml

```diff
@@ -131,151 +131,6 @@
-          volumes:
-          - name: artifacts
-            emptyDir: {}
-          - name: shared-tmp
-            emptyDir: {}
-          - name: cluster-profile
-            secret:
-              secretName: ${JOB_NAME_SAFE}-cluster-profile
-
-          containers:
-
-          # Once admin.kubeconfig exists, executes shared tests
-          - name: test
-            image: ${IMAGE_TESTS}
-            terminationMessagePolicy: FallbackToLogsOnError
-            resources:
-              requests:
-                cpu: 1
-                memory: 300Mi
-              limits:
-                memory: 4Gi
-            volumeMounts:
-            - name: shared-tmp
-              mountPath: /tmp/shared
-            - name: cluster-profile
-              mountPath: /tmp/cluster
-            - name: artifacts
-              mountPath: /tmp/artifacts
-            env:
-            - name: HOME
-              value: /tmp/home
-            - name: KUBECONFIG
-              value: /tmp/admin.kubeconfig
-            command:
-            - echo
-            - test
-            - CHANGED
-
-          # Runs an install
-          - name: setup
-            image: ${IMAGE_ANSIBLE}
-            terminationMessagePolicy: FallbackToLogsOnError
-            volumeMounts:
-            - name: shared-tmp
-              mountPath: /tmp
-            - name: cluster-profile
-              mountPath: /usr/share/ansible/openshift-ansible/inventory/dynamic/injected
-            env:
-            - name: INSTANCE_PREFIX
-              value: ${NAMESPACE}-${JOB_NAME_HASH}
-            - name: TYPE
-              value: ${CLUSTER_TYPE}
-            command:
-            - echo
-            - setup
-
-          # Performs cleanup of all created resources
-          - name: teardown
-            image: ${IMAGE_ANSIBLE}
-            terminationMessagePolicy: FallbackToLogsOnError
-            volumeMounts:
-            - name: shared-tmp
-              mountPath: /tmp/shared
-            - name: cluster-profile
-              mountPath: /usr/share/ansible/openshift-ansible/inventory/dynamic/injected
-            - name: artifacts
-              mountPath: /tmp/artifacts
-            env:
-            - name: INSTANCE_PREFIX
-              value: ${NAMESPACE}-${JOB_NAME_HASH}
-            - name: TYPE
-              value: ${CLUSTER_TYPE}
-            command:
-            - echo
-            - teardown
-  metadata:
-    creationTimestamp: null
-    labels:
-      ci.openshift.org/rehearse-pull: "1234"
-      created-by-pj-rehearse: "true"
-    name: rehearse-template-test-template-4b355dc3
-    namespace: test-namespace
-- data:
-    vars-origin.yaml: |
-      vars-origin.yaml
-    vars.yaml: |
-      changed
-  metadata:
-    creationTimestamp: null
-    labels:
-      ci.openshift.org/rehearse-pull: "1234"
-      created-by-pj-rehearse: "true"
-    name: rehearse-cluster-profile-test-profile-47224c86
-    namespace: test-namespace
```

Comment:
Do I understand correctly that this is no longer output because dry-run is now always "interfacing" with a single cluster, and this was only output because in the past there were more?

Reply:
Yup, exactly. Because I parallelized the config map creation it started to become indeterministic (in most but not all cases the goroutine we start first finishes first). Furthermore since we don't even keep the info on which cluster the thing gets created (its in all cases a client that just prints the serialized object) its also pretty useless.